From 8fafb6b7ab6d49f70699e50a2cc24d17c883a319 Mon Sep 17 00:00:00 2001
From: loopio <hxhao0613@outlook.com>
Date: Mon, 30 Oct 2017 22:51:22 +0800
Subject: [PATCH] =?UTF-8?q?=E6=B7=BB=E5=8A=A0=E5=8E=9Frepo=E4=BF=A1?=
 =?UTF-8?q?=E6=81=AF?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 README.md | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 37d6db35..c5c95dd4 100644
--- a/README.md
+++ b/README.md
@@ -1 +1,2 @@
-Hello linux-4.3.0-comment
\ No newline at end of file
+Hello linux-4.3.0-comment
+#这个repo是由《深入理解并行编程》译者谢宝友前辈在阅读内核代码时对内核代码进行的中文注释
-- 
2.25.1

